src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1746,14 +1746,14 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
 
         let mut hostflags = flags.clone();
         hostflags.push(format!("-Lnative={}", builder.test_helpers_out(compiler.host).display()));
-        hostflags.extend(builder.lld_flags(compiler.host));
+        hostflags.extend(linker_args(builder, compiler.host, LldThreads::No));
```

Comment:
`lld_flags` previously behave like `linker_flags`.
Is it intentional change to add `-Clinker` here?

Reply:
Good catch, that was not intentional. The linkers are set above it, so we probably don't want to also add them here.